_posts/2018-01-28-cashflow-pa-summary.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Cashflow PA
+subtitle: Summary and our timeline
+---
+
+In late September, 2017, we participated in the first [Code4PA hackathon](https://www.code4pa.tech/) in which we were tasked to use open state data to *increase transparency and efficiency for public engagement with the government*. We came up with Cashflow PA: a web application that maps campaign contributions, expenses, receipts, and debts filed for Pennsylvania state elections. It uses databases found on Pennsylvania's [open data portal](https://data.pa.gov/).
+
+Here's one of the mock-ups we created:
+
+![Cashflow PA screenshot](/img/cashflow-pa/cashflow-pa-screenshot.png)
+
+We pitched the app a month later and decided to keep working on it.
+
+## Our progress so far
+
+Here are some milestones we've already hit.
+
+* We've migrated a ton of data from a disparate collection of databases over to our own so that we can whip it into shape and serve it up fast.
+
+* We've implemented test-driven-development to build a sturdy app.
+
+* We've successfully implemented queries that return data to the views. Here's a shot of the working app on a local server:
+![Cashflow PA semi-working app](/img/cashflow-pa/cashflow-pa-working-map.png)
+
+## Where we're heading
+
+Timeline: we're hoping to roll out an MVP by February 19th (2018) and then presenting the (hopefully working) app to the PA Governor's office and a committee of executives on March 14th.
+
+We have several quality-of-life concerns to roll out for the app's basic functionality, and we're also planning to offer some insights into what we've found.
+
+Here's a screencap of our Trello board:
```

Comment:
If we wanted, we could make the Trello board public and link to it. Just an idea